rapid7_insightvm/plugin.spec.yaml

```
@@ -1653,7 +1658,7 @@ actions:
     output:
       id:
         description: Scan ID
-        title: Identifier of the resource created
+        title: Identifier of the Resource Created
```

```suggestion
        title: Scan ID
```
flip the description and title